content/blog/jt15s/2020-12-31_hello_from_the_haiku_promotion_team.md

```diff
@@ -0,0 +1,74 @@
+
+Apart from Auckland University, mmu_man, a member of the Haiku Community, also published a research paper about the interoperability of filesystem attributes.
+
+We aim to attract more research groups by promoting the uses of Haiku for research, and we also want to reach out to Auckland University and see if their UI research group is interested in coming back to Haiku to conduct any future research.
+
+### Businesses
+Businesses can benefit from Haiku as an operating system, and personally, I see Haiku as a lightweight alternative to the proprietary desktop operating systems businesses currently use. 
+
+An information sheet for businesses has been drafted, which gives businesses a rundown of Haiku and its advantages and disadvantages as a business-oriented operating system. However, to attract more businesses we must improve Haiku further, and hence we will delay marketing efforts for businesses until Haiku reaches a stable and mature stage. 
+
+### Partnerships with other Haiku-related sources
+If you haven't heard of them already, [BeSly](https://www.besly.de/index.php/de/) and [Haikunauts](https://haikunauts.home.blog/) are the two major third party Haiku sources. BeSly is a wiki with tips and tricks on using Haiku, whilst Haikunauts is a blog aimed at promoting Haiku as an OS. As mentioned above, we will work in partnership with these two sources and see what we can do together to help spread the word about Haiku.
+
+Additionally, we are also eager to work with any smaller blogs, websites or channels. If you run a blog, website, channel, or any other source, please contact me via the forums and let me know!
+
+### General
+I am thinking of running an advertisement in [Read the Docs](https://readthedocs.org/), a documentation website that serves a wide array of open-source projects. Read the Docs [allocates a portion of their advertising inventory to FOSS projects](https://docs.readthedocs.io/en/stable/advertising/ethical-advertising.html#community-ads) such as ourselves, hence this is a good opportunity to promote Haiku. What will go on the advertisement has not been decided as of now.
+
+scott_puopolo18 is working on a set of interview questions for our developers to give the public a glimpse into the world of Haiku development, as well as shining a light on the challenges and opportunities in an open-source project. The predetermined set of questions means that we don't have to write new questions for each developer. To save time, we can simply send them off and the developer can write their response.
+
+We are also planning to distribute more content via our often under-utilised social media platforms, as well as update them with current information (i.e. adding profile page art, applying for verification, updating or adding links etc.). Short-form posts on social media are what immediately comes to mind, although we are considering live-streams or videos if there is someone willing to help out.
```

Comment:
One thing to start with in the video side would be to collect and link the videos of all talks at various conferences (FOSDEM, RMLL, Capitole du Libre, BeGeistert, ...). Most were recorded but I think there is no easy central place to find them.